### ISSUE TITLE
bf luis:application:publish

### ISSUE THREAD

**Issue by felixvanleeuwen** [microsoft/botframework-cli#875]
## Versions
What CLI version are you using.
botframework-cli@4.10.0-preview.143194
What Nodejs version are you using
12.18.1
What command-line interpreters are you using
Powershell 6
What OS are you using
Windows 10

## Describe the bug
Give a clear and concise description of what the bug is.
bf luis:application:publish doesn't always publish.

## To Reproduce
1. Get a valid JSON to import;
2. Run this bit with appropriate parameters:
```
                Write-Host "> Uploading the $($luisApp.id)PortalExport.json file to LUIS because a change in version was detected..."
                bf luis:version:import `
                    --in $tempJsonPath `
                    --appId $luisApp.appId `
                    --versionId $portalExportObject.versionId `
                    --endpoint $config.cognitiveModels.endpoint `
                    --subscriptionKey $luisApp.subscriptionKey

                Write-Host "> Training the $($luisApp.id) model..."
                bf luis:train:run --appId $luisApp.appId `
                    --versionId $portalExportObject.versionId `
                    --endpoint $config.cognitiveModels.endpoint `
                    --subscriptionKey $luisApp.subscriptionKey

                Write-Host "> Publishing the $($luisApp.id) model..."
                bf luis:application:publish --endpoint $config.cognitiveModels.endpoint `
                    --subscriptionKey $luisApp.subscriptionKey `
                    --versionId $portalExportObject.versionId `
                    --appId $luisApp.appId
```

## Expected behavior
The imported model is trained an running in production.

## Screenshots
I just did 4 skills, 3 of which published just fine, the last one did not:
![image](https://user-images.githubusercontent.com/64589372/86484651-feecf280-bd56-11ea-91ea-3097abc77465.png)
Here's the output from one of the earlier ones:
![image](https://user-images.githubusercontent.com/64589372/86484681-11ffc280-bd57-11ea-98f3-eb40ab1bde03.png)
It doesn't look like there's any indication from the return value.

## Additional context

[bug]

**Comment by munozemilio** [comment]
@felixvanleeuwen just to confirm, whats the difference between the output when the model is trained and when its not?Have you tried using the --wait flag in the publish command?

**Comment by felixvanleeuwen** [comment]
Missed the --wait flag. I'll try it out and report back if I notice it happening again.

**Comment by felixvanleeuwen** [comment]
Had the --wait flag on and tried to update our skills. Some were not deployed to production:
![image](https://user-images.githubusercontent.com/64589372/87086559-141fc080-c232-11ea-9d06-356db410174a.png)
![image](https://user-images.githubusercontent.com/64589372/87086743-49c4a980-c232-11ea-9e23-9381cbed153b.png)

![image](https://user-images.githubusercontent.com/64589372/87086586-197d0b00-c232-11ea-9a83-e2c673d4863e.png)
![image](https://user-images.githubusercontent.com/64589372/87086793-58ab5c00-c232-11ea-9396-f3b8fd31d56c.png)

![image](https://user-images.githubusercontent.com/64589372/87086616-200b8280-c232-11ea-95e7-f364abd54e19.png)
![image](https://user-images.githubusercontent.com/64589372/87086847-695bd200-c232-11ea-89fb-c87a78e8f56e.png)

![image](https://user-images.githubusercontent.com/64589372/87086644-27329080-c232-11ea-9f95-1605428c94ce.png)
![image](https://user-images.githubusercontent.com/64589372/87086891-77115780-c232-11ea-9357-a6445d54eb5a.png)

Once again, the output from the publish does not imply a failed deploy.

**Comment by felixvanleeuwen** [comment]
I noticed a difference in how a successful train looks:
![image](https://user-images.githubusercontent.com/64589372/87088288-cfe1ef80-c234-11ea-83c8-14ed6c41f40e.png)

It seems it is the `--wait` flag that is not working as advertised?

**Comment by munozemilio** [comment]
@felixvanleeuwen could you please try the new nightly build from npm and see if you can repro with those bits?
```
npm i -g @microsoft/botframework-cli@next
```

**Comment by felixvanleeuwen** [comment]
We have two scripts that use this command. 

One, we ran last night, before updating to the new nightly. All models were successfully trained and published.

The other, we ran just now after updating to the new nightly. All models were successfully trained and published.

Reproducing this issue seems to rely on some fluke or our corporate proxy, which has been going through some changes recently.
